test/battle/crit_chance.c

```diff
@@ -1,11 +1,6 @@
 #include "global.h"
 #include "test/battle.h"
 
-ASSUMPTIONS
```

Comment:
I think you missed a few of these that fail with Gen 1 enabled, so should still have the ASSUME:
"Crit Chance: High crit rate increases the critical hit ratio by one stage"
"Crit Chance: Super Luck increases the critical hit ratio by one stage"